provisioning/playbook.yml

```diff
@@ -43,6 +43,7 @@
     - { role: geerlingguy.pimpmylog, when: '"pimpmylog" in installed_extras' }
     - { role: geerlingguy.daemonize, when: '"mailhog" in installed_extras' }
     - { role: geerlingguy.mailhog, when: '"mailhog" in installed_extras' }
+    - { role: franklinkim.newrelic, when: '"newrelic" in installed_extras' }
```

Comment:
As @oxyc mentioned, we need to add the role to the `requirements.yml` file as well.